api/app/signals/settings/base.py

```diff
@@ -426,7 +426,8 @@
     # Enabled the history_log based response, disables the signal_history_view based response
     'SIGNAL_HISTORY_LOG_ENABLED': os.getenv('SIGNAL_HISTORY_LOG_ENABLED', False) in TRUE_VALUES,
     'API_USE_QUESTIONNAIRES_APP_FOR_FEEDBACK': os.getenv('API_USE_QUESTIONNAIRES_APP_FOR_FEEDBACK', False) in TRUE_VALUES,  # noqa
-
+    'REPORTER_MAIL_HANDLED_NEGATIVE_CONTACT_ENABLED': os.getenv('REPORTER_MAIL_HANDLED_NEGATIVE_CONTACT_ENABLED', True) in TRUE_VALUES, # noqa
```

Comment:
Please remove these settings they are already defined a couple of lines bellow (Lines 444 and 447). Settings to be removed here are `REPORTER_MAIL_HANDLED_NEGATIVE_CONTACT_ENABLED`  and `REPORTER_MAIL_DISABLE_CONTACT_FEEDBACK_ALLOWS_CONTACT`.